apps/expo/index.js

```diff
@@ -1,8 +1,8 @@
-import { registerRootComponent } from 'expo';
```

Comment:
think you have linter misconfigured

Reply:
I will wait for @jmysliv with prettier and lint changes